org-roam.el

```diff
@@ -603,20 +603,45 @@ INFO is an alist containing additional information."
 ;;;; org-roam-link-face
 (defface org-roam-link
   '((t :inherit org-link))
-  "Face for org-roam link."
+  "Face for Org-roam links."
   :group 'org-roam-faces)
 
-(defface org-roam-backlink
+(defface org-roam-link-current
   '((t :inherit org-block))
-  "Face for org-roam backlinks in backlinks buffer"
+  "Face for Org-roam links pointing to the current buffer."
   :group 'org-roam-faces)
 
+(defun org-roam--in-buffer-p ()
+  "Return t if in the Org-roam buffer."
+  (string= (buffer-name)
+           org-roam-buffer))
+
+(defun org-roam--retrieve-link-path (&optional pom)
+  "Retrieve the path of the link at point or POM.
+POM can be a position in the current buffer or a marker."
```

Comment:
```suggestion
(defun org-roam--retrieve-link-path (&optional point-or-marker)
  "Retrieve the path of the link at POINT-OR-MARKER.
  If nil, POINT-OR-MARKER is set to the current point."
```

No need to abbrieviate here.

Reply:
It's actually what Org-mode uses in some of their commands:
```el
(defmacro org-with-point-at (pom &rest body)
  "Move to buffer and point of point-or-marker POM for the duration of BODY."
  (declare (debug (form body)) (indent 1))
  (org-with-gensyms (mpom)
    `(let ((,mpom ,pom))
       (save-excursion
	 (when (markerp ,mpom) (set-buffer (marker-buffer ,mpom)))
	 (org-with-wide-buffer
	  (goto-char (or ,mpom (point)))
	  ,@body)))))
```

I don't mind changing it, I'll leave it up to you.

Reply:
hmm maybe follow their docstring. I was initially confused by the "point or POM"

Reply:
I think it's not clear.  Do you prefer 36199f15bd8dda34961404e8a7e2562a88a1b6bb?